tools/depends/configure.ac

```diff
@@ -73,13 +73,15 @@ AC_ARG_WITH([sdk-path],
 
 AC_ARG_WITH([sdk],
   [AS_HELP_STRING([--with-sdk],
-  [specify sdk platform version (optional for android). default is android-24])],
-  [use_sdk=$withval])
+  [specify sdk platform version (optional for android). default is android-26])],
+  [use_sdk=$withval],
+  [use_sdk=android-26])
```

Comment:
`use_sdk` isn't only android specific (used for darwin too) so don't set an android default.

Reply:
@rechi Yes for defaultenv, No for Readme.android, the android-21 was simple double.

In the lines you marked is android-21 already included, in the lines where you make the toolchain they simply were wrong. So there is no need to change README as well.

Reply:
the use_sdk sure, already mentioned here a few seconds after PR...